experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
```

Comment:
+1 Let's make sure we are keeping parody with the existing FluidContainer interface.

Reply:
@sumedhb1995 , both were added. I like the static create because it consolidates the rootDataObject within the FluidContainer where it is defined instead of putting the responsibility on the caller to know how to get the root.